test/show_test.jl

```diff
@@ -6,7 +6,7 @@ end
 
 let kws = [], msgs =[]
     ds = DummyStream(IOBuffer(), 0, 0)
-    for s = 0:9, h in [Base.C_NULL, Ptr{UInt64}(3)]
+    for s = 0:9, h in [Base.C_NULL, Ptr{Cvoid}(3)]
```

Comment:
I'm just making a note as I study your changes.
WebSockets uses Base.LibuvStream, which depends on Base.LibuvServer, which  specifically requires handle to be of type Ptr{Cvoid} after Julia 1.6. Cvoid to aliased to the Nothing type.